api_tests/src/shared.ts

```diff
@@ -364,10 +364,13 @@ export async function getUnreadCount(
   return api.getUnreadCount();
 }
 
-export async function getReplies(api: LemmyHttp): Promise<GetRepliesResponse> {
+export async function getReplies(
+  api: LemmyHttp,
+  unread_only: boolean = false,
+): Promise<GetRepliesResponse> {
   let form: GetReplies = {
     sort: "New",
-    unread_only: false,
+    unread_only: unread_only,
```

Comment:
Surprised whatever IDE you're using isn't telling you that this can be simplified to just `unread_only,`

Reply:
just started with RustRover after Jetbrains just released a free version for non-commercial use.
might be better with some additional plugins.

Reply:
Might need to install a typescript plugin, not sure.

Reply:
updated.
typescript plugin is already installed, but it's not flagging this :shrug: